plugins/aws/workspaces/workspacesOperationalState.js

```diff
@@ -0,0 +1,52 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Workspaces Operational State',
+    category: 'Workspaces',
+    description: 'Ensures instances are healthy on Workspaces',
+    more_info: 'Checking the workspaces instances state are healthy or not?',
+    link: 'https://docs.aws.amazon.com/workspaces/latest/adminguide/cloudwatch-metrics.html',
+    recommended_action: 'Ensures that workspaces instance is respond to service health checks',
+    apis: ['WorkSpaces:describeWorkspaces'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+
+        var regions = helpers.regions(settings);
+
+        async.each(regions.workspaces, function(region, rcb) {
+            var listWorkspaces = helpers.addSource(cache, source, ['workspaces', 'describeWorkspaces', region, 'data']);
```

Comment:
rename the variable to be describeWorkspaces as that is the api we are calling and getting the data here.

Reply:
done